pkg/filter/proxy/server_test.go

```diff
@@ -365,6 +365,9 @@ func TestDynamicService(t *testing.T) {
 	}
 
 	if !reflect.DeepEqual(wantStatic, s.static) {
-		t.Fatalf("want: %+v\ngot :%+v\n", wantStatic, s.static)
+		wantStatic.servers[0], wantStatic.servers[1] = wantStatic.servers[1], wantStatic.servers[0]
```

Comment:
This solution is not general enough, what if it adds more servers. I think we should sort the `want` and `got` by URL then compare them.

Reply:
good idea!